tests/unit/api/objectReplicationMD.js

```diff
@@ -77,7 +77,17 @@ const emptyReplicationMD = {
     role: '',
     storageType: '',
     dataStoreVersionId: '',
-    isNFS: null,
+    isNFS: undefined,
+};
+const emptyReplicationMDUndef = {
```

Comment:
I don't think we put replicationInfo with `isNFS: undefined` : same (empty) fields are indeed not returned anymore (change in arsenal), but we did not change the way we write the metadata, so the "input" parameter of the tests should not change


```suggestion
    isNFS: null,
};
const expectedEmptyReplicationMD = {
```

Reply:
The test is failing otherwise waiting for undefined instead of nul

Reply:
> we did not change the way we write the metadata, so the "input" parameter of the tests should not change

if the test fail, it is probably because we at some point compare to `emptyReplicationMD` instead of `emptyReplicationMDUndef` (which would be better renamed `expectedEmptyReplicationMD` I think)